packages/nvidia_nat_weave/tests/test_fastapi_plugin_worker.py

```
@@ -0,0 +1,134 @@
+        temp_dir = tempfile.gettempdir()
+        os.environ["NAT_CONFIG_FILE"] = os.path.join(temp_dir, "dummy_nat_config.yml")
+    yield
+
+
+@pytest.fixture(name="mock_weave", autouse=True)
+def fixture_mock_weave(monkeypatch):
+    """Mock weave.init and weave client context to avoid authentication issues in unit tests."""
+    mock_weave_client = MagicMock()
+
+    # Mock weave.init
+    monkeypatch.setattr("weave.init", lambda *args, **kwargs: mock_weave_client, raising=False)
```

Ruff (0.14.8)

Unused lambda argument: `args`

(ARG005)